x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/SearchableSnapshotActionIT.java

```diff
@@ -449,23 +450,32 @@ public void testConvertingSearchableSnapshotFromFullToPartial() throws Exception
 
     @SuppressWarnings("unchecked")
     public void testConvertingPartialSearchableSnapshotIntoFull() throws Exception {
-        String index = "myindex-" + randomAlphaOfLength(4).toLowerCase(Locale.ROOT);
+        String index = "myindex-" + randomAlphaOfLength(4).toLowerCase(Locale.ROOT) +"-000001";
         createSnapshotRepo(client(), snapshotRepo, randomBoolean());
-        createPolicy(client(), policy, null, null,
-            new Phase("cold", TimeValue.ZERO,
-                singletonMap(SearchableSnapshotAction.NAME, new SearchableSnapshotAction(snapshotRepo, randomBoolean(),
-                    MountSearchableSnapshotRequest.Storage.SHARED_CACHE))),
+        createPolicy(client(), policy,
+            new Phase("hot", TimeValue.ZERO,
```

Comment:
Piggybacked the hot -> frozen searchable_snapshot actions on this test (as it wasn't tested anywhere)